DEVELOPMENT.md

```diff
@@ -0,0 +1,109 @@
+# Developing Buzz About
+
+Before you can start working on Buzz About, there are a few things you'll have to set up on your computer.
+
+**Table of Contents**
+
+- [Tell `npm` how to talk to GitHub Packages](#tell-npm-how-to-talk-to-github-packages)
+  - [Generating the personal access token](#generating-the-personal-access-token)
+  - [Giving that personal access token to `npm`](#giving-that-personal-access-token-to-npm)
+- [Installing the Angular command-line interface](#installing-the-angular-command-line-interface)
+- [Setting up Firebase](#setting-up-firebase)
+  - [Installing the command-line interface](#installing-the-command-line-interface)
+  - [Running the emulator](#running-the-emulator)
+
+## Tell `npm` how to talk to GitHub Packages
+
+GitHub Packages is where we keep all of the images for the website, along with some data about flowers and bees. 
+
+For `npm` read from GitHub Packages, you'll need to provide it with a personal access token.
+
+### Generating the personal access token
+
+You can create a personal access token by going to <https://github.com/settings/tokens> and clicking "Generate new token". Under "Scopes", you'll want to check `repo`, `read:packages`, `write:packages`, and `delete:packages`:
+
+![The "New personal access token" page on GitHub, with those four scopes checked.][i:New personal access token]
+
+Then, you can click "Generate token" at the bottom of the page. You'll get a token that looks like `745b0dcc742dedf87c67ff075f65eb87295e205f`. Keep that around.
+
+### Giving that personal access token to `npm`
+
+Next, open the command-line and run
+
+```sh
+npm login --registry=https://npm.pkg.github.com
+```
+
+ * When it asks for a username, enter your GitHub username.
+ * When it asks for a password, paste in the personal access token (***not*** your GitHub password).
+ * When it asks for your email, enter your GitHub email. 
+
+![Powershell after running "npm login --registry=https://npm.pkg.github.com". The output "Logged in as helloworld12321demo on https://npm.pkg.github.com/" is shown.][i:Logging in to npm]
```

Comment:
I don't know if a screenshot for this is necessary. If we do keep it we should make the window shorter if possible.

Reply:
Roger that!

Reply:
I think I'm going to delete that screenshot, given that it's a bit awkward anyway. (I was using a throw-away GitHub account, so my username and email look like gobbledygook.)